### DISCUSSION TITLE
Take out certain hours from analysis

### DISCUSSION THREAD

**Discussion by ziqbal99** [spyder-ide/qtpy#435] [Show and tell]
-how to write a code to eliminate certain hours of data from analysis like I am running a report from 6am to 5pm. How can I skip 7-9 am data from report. Can someone provide example of python code?

**Comment by ccordoba12** [comment]
Hey @ziqbal99, thanks for reporting. Unfortunately, this is not a support website but a place to report errors or suggestions about the Qtpy library.

Instead, please try to open a question in Stack overflow about it:

https://stackoverflow.com/
